apph-back/src/main/java/com/viseo/apph/dto/JwtResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.viseo.apph.dto;
+
+public class JwtResponse {
+    String token;
+
+    public JwtResponse(String token) {
+        this.token = token;
+    }
+
+    public String getToken() {
+        return token;
+    }
+
+    public void setToken(String token) {
+        this.token = token;
+    }
```

Comment:
Je crois que nos conventions veulent que les setter renvoient `this` pour pouvoir les enchaîner (je crois que c'est le patron de conception 'Builder')